### ISSUE TITLE
--download-sections usage

### ISSUE THREAD

**Issue by honx4amiga** [yt-dlp/yt-dlp#5955]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2023.01.02** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

i can't download sections of youtube videos anymore.

in the past i used to use yt-dlp command looking like following for download sections of mp3:
yt-dlp --extract-audio --audio-format mp3 --no-mtime --output "%%(title)s.%%(ext)s" --download-sections "*0:09-4:29" GQSUIRDs6VE
but now this doesn't work anymore. the line shown above is the exact example of that song i tried to download today: https://www.youtube.com/watch?v=GQSUIRDs6VE
i tried to skip that wind sound at the beginning (9 seconds) and the silent (no sound) part at the end of that song.

now i'm getting following error:
"[NULL @ 00000000038bd2c0] Requested output format 'weba' is not a suitable outpu
t format
file:Red House Painters - All Mixed Up [GQSUIRDs6VE].weba.part: Invalid argument

ERROR: ffmpeg exited with code 1"
i attach full verbose log (-vU) argument.

what am i doing wrong nowadays? why can't i download sections anymore for mp3 output?

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['--extract-audio', '--audio-format', 'mp3', '--no-
mtime', '--output', '%%(title)s.%%(ext)s', '--download-sections', '*0:09-4:29',
'GQSUIRDs6VE', '-vU']
[debug] User config: []
[debug] System config: []
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8 (No VT), erro
r utf-8 (No VT), screen utf-8 (No VT)
[debug] yt-dlp version 2023.01.02 [d83b0ad] (win32_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-7-6.1.7601-SP1 (OpenSSL 1.
1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 5.0-essentials_build-www.gyan.dev (setts)
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07,
 mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1754 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releas
es/latest
Latest version: 2023.01.02, Current version: 2023.01.02
yt-dlp is up to date (2023.01.02)
[youtube] Extracting URL: GQSUIRDs6VE
[youtube] GQSUIRDs6VE: Downloading webpage
[youtube] GQSUIRDs6VE: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec
:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source
, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr
, abr, asr, vext, aext, hasaud, id
[info] GQSUIRDs6VE: Downloading 1 format(s): 251
[info] GQSUIRDs6VE: Downloading 1 time ranges: 9.0-269.0
[debug] Invoking ffmpeg downloader on "https://rr1---sn-puigxxc-8pxl.googlevideo
.com/videoplayback?expire=1672889678&ei=7vC1Y_GnII-M6dsP56mwSA&ip=78.142.74.185&
id=o-ACFcTBQmsMRtMR_wjNj4JVrEWiWaEKWU5pTP3RA-f3Vu&itag=251&source=youtube&requir
essl=yes&mh=bN&mm=31%2C29&mn=sn-puigxxc-8pxl%2Csn-4g5edndl&ms=au%2Crdu&mv=m&mvi=
1&pl=21&gcr=at&initcwndbps=1603750&spc=zIddbEPy-YBhzbbyIBNmxTa7-9Es1-s&vprv=1&sv
puc=1&mime=audio%2Fwebm&gir=yes&clen=4593478&dur=282.401&lmt=1496518241952333&mt
=1672867754&fvip=1&keepalive=yes&fexp=24007246&beids=24239129&c=ANDROID&sparams=
expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cgcr%2Cspc%2Cvprv%2Csvpuc%2C
mime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAOuEoFO7bPuLo0wcPlK51wg2VcA6t6EHl0
xYyoiC5iovAiEAvxxczRHkSjx9hp02MAoYMwUBJ795gNCyMTR6ibJWbeM%3D&lsparams=mh%2Cmm%2C
mn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAM3ihQlo9pYkElXJSK1bSSbOy
M4I1MPu_HQ7am6rjMWEAiA6S4FXs2gHKU6Atx-8i7THzh9ZnIi42Y6zgQpHLFvDvA%3D%3D"
[download] Destination: %(title)s.%(ext)s
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers "User-Agent: M
ozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)
Chrome/90.0.4430.212 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
" -ss 9.0 -t 260.0 -i "https://rr1---sn-puigxxc-8pxl.googlevideo.com/videoplayba
ck?expire=1672889678&ei=7vC1Y_GnII-M6dsP56mwSA&ip=78.142.74.185&id=o-ACFcTBQmsMR
tMR_wjNj4JVrEWiWaEKWU5pTP3RA-f3Vu&itag=251&source=youtube&requiressl=yes&mh=bN&m
m=31%2C29&mn=sn-puigxxc-8pxl%2Csn-4g5edndl&ms=au%2Crdu&mv=m&mvi=1&pl=21&gcr=at&i
nitcwndbps=1603750&spc=zIddbEPy-YBhzbbyIBNmxTa7-9Es1-s&vprv=1&svpuc=1&mime=audio
%2Fwebm&gir=yes&clen=4593478&dur=282.401&lmt=1496518241952333&mt=1672867754&fvip
=1&keepalive=yes&fexp=24007246&beids=24239129&c=ANDROID&sparams=expire%2Cei%2Cip
%2Cid%2Citag%2Csource%2Crequiressl%2Cgcr%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Ccle
n%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAOuEoFO7bPuLo0wcPlK51wg2VcA6t6EHl0xYyoiC5iovAiEAvx
xczRHkSjx9hp02MAoYMwUBJ795gNCyMTR6ibJWbeM%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cm
vi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAM3ihQlo9pYkElXJSK1bSSbOyM4I1MPu_HQ7am6rj
MWEAiA6S4FXs2gHKU6Atx-8i7THzh9ZnIi42Y6zgQpHLFvDvA%3D%3D" -c copy -f weba "file:%
(title)s.%(ext)s.part"
ffmpeg version 5.0-essentials_build-www.gyan.dev Copyright (c) 2000-2022 the FFm
peg developers
  built with gcc 11.2.0 (Rev5, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32thr
eads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --e
nable-libxml2 --enable-gmp --enable-lzma --enable-zlib --enable-libsrt --enable-
libssh --enable-libzmq --enable-avisynth --enable-sdl2 --enable-libwebp --enable
-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg
--enable-libvpx --enable-libass --enable-libfreetype --enable-libfribidi --enabl
e-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm -
-enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va
--enable-dxva2 --enable-libmfx --enable-libgme --enable-libopenmpt --enable-libo
pencore-amrwb --enable-libmp3lame --enable-libtheora --enable-libvo-amrwbenc --e
nable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --ena
ble-libvorbis --enable-librubberband
  libavutil      57. 17.100 / 57. 17.100
  libavcodec     59. 18.100 / 59. 18.100
  libavformat    59. 16.100 / 59. 16.100
  libavdevice    59.  4.100 / 59.  4.100
  libavfilter     8. 24.100 /  8. 24.100
  libswscale      6.  4.100 /  6.  4.100
  libswresample   4.  3.100 /  4.  3.100
  libpostproc    56.  3.100 / 56.  3.100
[tcp @ 000000000036b300] Starting connection attempt to 95.143.84.104 port 443
[tcp @ 000000000036b300] Successfully connected to 95.143.84.104 port 443
Input #0, matroska,webm, from 'https://rr1---sn-puigxxc-8pxl.googlevideo.com/vid
eoplayback?expire=1672889678&ei=7vC1Y_GnII-M6dsP56mwSA&ip=78.142.74.185&id=o-ACF
cTBQmsMRtMR_wjNj4JVrEWiWaEKWU5pTP3RA-f3Vu&itag=251&source=youtube&requiressl=yes
&mh=bN&mm=31%2C29&mn=sn-puigxxc-8pxl%2Csn-4g5edndl&ms=au%2Crdu&mv=m&mvi=1&pl=21&
gcr=at&initcwndbps=1603750&spc=zIddbEPy-YBhzbbyIBNmxTa7-9Es1-s&vprv=1&svpuc=1&mi
me=audio%2Fwebm&gir=yes&clen=4593478&dur=282.401&lmt=1496518241952333&mt=1672867
754&fvip=1&keepalive=yes&fexp=24007246&beids=24239129&c=ANDROID&sparams=expire%2
Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cgcr%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cg
ir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAOuEoFO7bPuLo0wcPlK51wg2VcA6t6EHl0xYyoiC5i
ovAiEAvxxczRHkSjx9hp02MAoYMwUBJ795gNCyMTR6ibJWbeM%3D&lsparams=mh%2Cmm%2Cmn%2Cms%
2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAM3ihQlo9pYkElXJSK1bSSbOyM4I1MPu_
HQ7am6rjMWEAiA6S4FXs2gHKU6Atx-8i7THzh9ZnIi42Y6zgQpHLFvDvA%3D%3D':
  Metadata:
    encoder         : google
  Duration: 00:04:42.40, start: -0.007000, bitrate: 130 kb/s
  Stream #0:0(eng): Audio: opus, 48000 Hz, stereo, fltp, delay 312 (default)
[NULL @ 000000000380b2c0] Requested output format 'weba' is not a suitable outpu
t format
file:%(title)s.%(ext)s.part: Invalid argument
[AVIOContext @ 0000000003711640] Statistics: 16384 bytes read, 0 seeks


ERROR: ffmpeg exited with code 1
  File "yt_dlp\__main__.py", line 17, in <module>
  File "yt_dlp\__init__.py", line 970, in main
  File "yt_dlp\__init__.py", line 960, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3353, in download
  File "yt_dlp\YoutubeDL.py", line 3328, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1486, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1497, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1594, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1653, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2767, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3231, in process_info
  File "yt_dlp\YoutubeDL.py", line 2954, in dl
  File "yt_dlp\downloader\common.py", line 444, in download
  File "yt_dlp\downloader\external.py", line 76, in real_download
  File "yt_dlp\YoutubeDL.py", line 1007, in report_error
  File "yt_dlp\YoutubeDL.py", line 936, in trouble
```

**Comment by bashonly** [comment]
This is due to a bug introduced in version 2023.01.02 where webm audio is incorrectly extracted with a `.weba` extension. It has been fixed in d80ca5deaa46db6e498399bb04a72a4c10ee8e22, and will be fixed in the next release.

A workaround for now is to use `-S "ext"` to prefer mp4 formats over webm.

If the workaround is not acceptable and you don't want to wait for the next release, you can [install the master branch with PIP](https://github.com/yt-dlp/yt-dlp/wiki/Installation#with-pip), or try the [unofficial daily builds](https://github.com/ytdl-patched/yt-dlp)

**Comment by honx4amiga** [comment]
thanks, the workaround (adding -S "ext" to command) worked for me.

there was just a warning during extraction:
"[ipod @ 0000000003704840] Warning, extension is not .m4a nor .m4v Quicktime/Ipod
 might not play the file"
since i don't own any ipod, i think i can ignore that warning. :) i got what i wanted, i got a mp3 file missing these unwanted parts at beginning and end.

so i can wait for next official release, there's no need for daily builds. :)
